.travis.yml

```diff
@@ -38,6 +38,12 @@ compiler:
   - clang
 env:
   matrix:
+  - Q_OR_C_MAKE=cmake
+    QT_VERSION=59
+    WITH_FONTS=NO # actually Qt 5.9, non-default without fonts
```

Comment:
Temporary extension, right? I'd rather just test with font inclusion in for the sake of reasonable-ish build times.

Reply:
Yeah, just for testing - will take the CI build matrix back down before merge.

Reply:
Reverted now!